src/main/java/gitflow/actions/FinishBugfixAction.java

```diff
@@ -62,12 +62,12 @@ public void runAction(final Project project, final String bugfixName){
         new Task.Backgroundable(myProject,"Finishing bugfix "+bugfixName,false){
             @Override
             public void run(@NotNull ProgressIndicator indicator) {
-                GitCommandResult result =  myGitflow.finishBugfix(myRepo, bugfixName, errorLineHandler);
-
                 //get the base branch for this bugfix
                 final String baseBranch = gitflowConfigUtil.getBaseBranch(branchUtil.getPrefixBugfix()+bugfixName);
 
-                if (result.success()) {
+				GitCommandResult result =  myGitflow.finishBugfix(myRepo, bugfixName, errorLineHandler);
+
```

Comment:
Please fix the white space issues

Reply:
considering that there was no activity on this PR, I think you'd have to change that yourself.